crates/re_types_builder/src/codegen/cpp/mod.rs

```diff
@@ -1892,8 +1893,22 @@ fn quote_fqname_as_type_path(includes: &mut Includes, fqname: &str) -> TokenStre
     quote!(#expr)
 }
 
+fn get_examples(docs: &Docs) -> Vec<String> {
+    // `cpp` examples are not required for now, so just default to empty vec
+    crate::codegen::get_examples(docs, "cpp", &["```cpp,ignore"], &["```"]).unwrap_or_default()
+}
+
 fn quote_docstrings(docs: &Docs) -> TokenStream {
-    let lines = crate::codegen::get_documentation(docs, &["cpp", "c++"]);
+    let mut lines = crate::codegen::get_documentation(docs, &["cpp", "c++"]);
+
+    let examples = get_examples(docs);
+    if !examples.is_empty() {
+        lines.push(String::new());
+        lines.push("## Example".into());
```

Comment:
Let's go the extra mile here and for the other languages:
```suggestion
        if examples.len() == 1 {
            lines.push("## Example".into());
        } else {
            lines.push("## Examples".into());
        }
```